app/src/main/res/drawable/partly_cloudy_day.xml

```diff
@@ -0,0 +1,4 @@
+<vector android:height="48dp" android:viewportHeight="512.0"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.partly_cloudy_day` appears to be unused
 Unused resources make applications larger and slow down builds.